include/FockSpace/BaseFockSpace.hpp

```diff
@@ -36,6 +37,17 @@ class BaseFockSpace {
     virtual ~BaseFockSpace() = 0;
 
 
+    // GETTERS
+    size_t get_dimension() const { return dim; }
+    size_t get_K() const { return K; }
+
+
+    // PUBLIC METHODS
+    /**
+     *  Creates a Hartree-Fock guess
```

Comment:
Document that the HF wave function is an expansion with only 1 Slater determinant